src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let content;
+
+  if (!person.isMarried) {
+    content
+      = (
+        <p className="Person__partner">
+          `I am not married`
+        </p>
+      );
+  } else if (person.sex === 'm') {
+    content = <p className="Person__partner">{`${person.partnerName} is my wife`}</p>;
+  } else {
+    content
+      = (
+        <p className="Person__partner">
+          {`${person.partnerName} is my husband`}
+        </p>
+      );
+  }
```

Comment:
redundant else if.
enough to make a couple teranry and only for text.

```suggestion
const personPartnerType = sex === 'm' : 'wire' : 'husband';
const personParner = isMarried
  ? `${person.partnerName} is my ${personPartnerType}`
  : 'I am not married'
```
and use personPartner in jsx